src/middleware/signOutUserBanner.middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionInterfaces'
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+
+import SessionService from 'app/services/session/session.service'
+
+export default function SignOutUserBannerMiddleware(sessionService: SessionService): RequestHandler {
```

Comment:
This middleware will need to occur after the user has authenticated and the session has been created